Class Tweakers: Enum Extensions (#567)
FabricMC/fabric-docs: develop/class-tweakers/index.md

The file must have the following header as its first line:

```classtweaker
classTweaker  v1  official
```

> Some features may require you to use a version number higher than `v1`, which will be specified in their dedicated pages.

Reviewer: ```suggestion
Some features may require a version higher than `v1` - that will be mentioned in their respective pages.
```
